tests/test_api.py

```diff
@@ -474,13 +474,6 @@ def test_handle_smart_errors(self):
                 "message": NotFound.description,
             }) + "\n")
 
-        with app.test_request_context("/fOo"):
```

Comment:
this whole test case can just be removed. 404 handling is the only 'smart error' capability